### ISSUE TITLE
Make the default location and name of saved responses configurable in the settings

### ISSUE THREAD

**Issue by dralley** [Huachao/vscode-restclient#186]
I would prefer to be able to set my default save location to, say, ```~/Downloads``` instead of ```~/``` and also that the files weren't hidden

**Comment by Huachao** [comment]
@dralley @UsulPro I have implemented this feature by opening a saving dialogue when clicking `Save Response` and `Save Response Body`. So that you can decide where to store the file as you wish, and the default location in the dialogue will keep unchanged.

**Comment by Huachao** [comment]
@dralley @UsulPro you can try the latest version to verify.

**Comment by Huachao** [comment]
@UsulPro I totally agree, I will reopen this issue.

**Comment by samthegolden** [comment]
Any update on this? It's open for 2 years now.

**Comment by Huachao** [comment]
@samthegolden no, if you have some interest, it's warmly welcomed to contribute a PR
